### PR TITLE
Search book( pagination + modifi css search)

### PR DESCRIPTION
**Pull request by quytho**
## Checklist
- [ ] This is bug/feature (delete incorrect one)
- [ ] Title including ticket number and title

## What this PR do?
Explain clearly and briefly what this PR does. Make a meaningful sentence instead of "this PR solve ticket #xxx"

## Context/Background
More information on the issue that you want to provide, related issues and PRs you have to write information to ask orally or in informal place too.

## Screenshots / Demo & Instruction for testing (if applicable)
image
image

- link to demo
- instruction on how to test this

## References
- Blog posts / web pages relevant to this issue (which you get your solution / idea from and it may help others to understand the issue quickly)
- Do we need update our knowledge base / project wiki?

### CODE REVIEW

File: app/controllers/books_controller.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class BooksController < ApplicationController
   def index
-    @books = Book.search(params)
+    @books = Book.search(params).paginate(page: 1, per_page: 11)
```
Comment (rikai-hai) [inline app/controllers/books_controller.rb]: tại sao lại set cứng hè ?

File: app/helpers/books_helper.rb [hunk 2]
```diff
@@ -1,2 +1,8 @@
 module BooksHelper
+  def find_author(book)
+    author = Author.find(book.author_id)
+  end
```
Comment (rikai-hai) [inline app/helpers/books_helper.rb]: cách space

File: app/helpers/books_helper.rb [hunk 3]
```diff
@@ -1,2 +1,8 @@
 module BooksHelper
+  def find_author(book)
+    author = Author.find(book.author_id)
```
Comment (rikai-hai) [inline app/helpers/books_helper.rb]: ko đc query trong view

File: app/helpers/books_helper.rb [hunk 4]
```diff
@@ -1,2 +1,8 @@
...
+  end
+  def find_rate(book)
+    rate = Rate.find(book.id)
```
Comment (rikai-hai) [inline app/helpers/books_helper.rb]: ko đc query trong view

File: app/views/books/_item-book.slim [hunk 5]
```diff
@@ -1,19 +1,25 @@
...
+          | Tác giả: 
+          = find_author(book).name
+        p.book-amount 
+          | Sô lượng:
+          = book.amount
+        p.book-rate
+          | Đánh giá:
+          = find_rate(book).star
+        = link_to "follow",'#',class: "follow"          
+        = link_to "Details", "#",class: "details"
```
Comment (rikai-hai) [inline app/views/books/_item-book.slim]: chuyển thành 1 partial
sau đó ` render @books`

File: app/views/books/_item-book.slim [hunk 6]
```diff
@@ -1,19 +1,25 @@
...
+= will_paginate @books
```
Comment (rikai-hai) [inline app/views/books/_item-book.slim]: cho center vào

**Comment by quytho** [comment]
image

**Pull request merged**